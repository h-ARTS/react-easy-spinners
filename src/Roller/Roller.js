import React from 'react'
import styles from './styles.css'

const Roller = (props) => (
  <>
    <style>${`${styles}`}</style>
    <div className="lds-roller">
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
    </div>
  </>
)

export default Roller
