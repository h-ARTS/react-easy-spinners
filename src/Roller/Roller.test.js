import React from 'react'
import renderer from 'react-test-renderer'
import Roller from './Roller'

describe('Roller component', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Roller />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
