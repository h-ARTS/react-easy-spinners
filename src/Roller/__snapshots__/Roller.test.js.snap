// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Roller component should render correctly 1`] = `
<div
  className="lds-roller"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
